src/ui/simulator/windows/options/advanced/advanced.cpp

```diff
@@ -930,7 +930,7 @@ void AdvancedParameters::onSelectNumberOfCoresLevel(Data::NumberOfCoresMode ncMo
     {
         study->parameters.nbCores.ncMode = ncMode;
         // Force refresh for study->nbYearsParallelRaw
-        study->getNumberOfCores(false, 1 /* ignored */);
+        study->getNumberOfCores(false, false, 1 /* ignored */);
```

Comment:
**Question** : why do we suppose that the **getNumberOfCores**'s second argument (which is new here) is necessarily **false** ?

Reply:
I don't know. Honestly it feels like the lesser issue of "getNumberOfCores"
- It is in the study
- It's called "get" but doesn't return anything
- Last parameter is ignored if the first one is false

Reply:
At least we could rename **getNumberOfCores** into **updateNumberOfCores**